modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -7695,6 +7797,11 @@ private JSONObject _toFileEntryJSONObject(
 	@Inject
 	private ObjectEntryLocalService _objectEntryLocalService;
 
+	@Inject(
+		filter = "object.entry.manager.storage.type=" + ObjectDefinitionConstants.STORAGE_TYPE_DEFAULT
+	)
+	private ObjectEntryManager _objectEntryManager;
```

Comment:
@devpedrotavares if you prefer to use the default `ObjectEntryManager` instead, then you should move the created test to `DefaultObjectEntryManagerImplTest`

Reply:
Okay. I moved it because we need the formatting that is done by `DefaultObjectEntryManagerImpl`